swagger.json

```diff
@@ -0,0 +1,1344 @@
+          "items": {
+            "$ref": "#/definitions/DynamicElement"
+          },
+          "x-go-name": "DynamicElements"
+        },
+        "href": {
+          "type": "string",
+          "x-go-name": "Href"
+        },
+        "last_advertised_timestamp": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "LastAdvertisedTimestamp"
+        },
+        "namespace": {
+          "type": "string",
+          "x-go-name": "Namespace"
+        },
+        "policy": {
+          "$ref": "#/definitions/PolicyTableSlice"
+        },
+        "unit": {
+          "type": "string",
+          "x-go-name": "Unit"
+        },
+        "version": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "Version"
+        }
+      },
+      "x-go-package": "github.com/intelsdi-x/snap/mgmt/rest/v2"
+    },
+    "Plugin": {
+      "type": "object",
+      "title": "Plugin represents a plugin type definition.",
+      "properties": {
+        "config_policy": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PolicyTable"
+          },
+          "x-go-name": "ConfigPolicy"
+        },
+        "hitcount": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "HitCount"
+        },
+        "href": {
+          "type": "string",
+          "x-go-name": "Href"
+        },
+        "id": {
+          "type": "integer",
+          "format": "uint32",
+          "x-go-name": "ID"
+        },
+        "last_hit_timestamp": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "LastHitTimestamp"
+        },
+        "loaded_timestamp": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "LoadedTimestamp"
+        },
+        "name": {
+          "type": "string",
+          "x-go-name": "Name"
+        },
+        "pprof_port": {
+          "type": "string",
+          "x-go-name": "PprofPort"
+        },
+        "signed": {
+          "type": "boolean",
+          "x-go-name": "Signed"
+        },
+        "status": {
+          "type": "string",
+          "x-go-name": "Status"
+        },
+        "type": {
+          "type": "string",
+          "x-go-name": "Type"
+        },
+        "version": {
+          "type": "integer",
+          "format": "int64",
+          "x-go-name": "Version"
+        }
+      },
+      "x-go-package": "github.com/intelsdi-x/snap/mgmt/rest/v2"
+    },
+    "PolicyTable": {
+      "$ref": "#/definitions/RuleTable"
+    },
+    "PolicyTableSlice": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/RuleTable"
+      },
+      "x-go-package": "github.com/intelsdi-x/snap/mgmt/rest/v2"
+    },
+    "ProcessWorkflowMapNode": {
+      "type": "object",
+      "required": [
+        "plugin_name"
+      ],
+      "properties": {
+        "config": {
+          "description": "TODO processor config",
```

Comment:
This one should not have such description. Of course, this points to that [this line](https://github.com/intelsdi-x/snap/pull/1599/files#diff-e1f961b73cb3ace6fed8796b07b717fdR293) should be changed.